code/src/vs/workbench/contrib/remote/browser/che/remote.ts

```diff
@@ -53,7 +53,7 @@ export class CheDisconnectionHandler {
 	}
 
 	private restartWorkspaceHandler = () => {
-		this.devWorkspaceAssistant.restartWorkspace();
+		this.devWorkspaceAssistant.startWorkspace();
```

Comment:
At the first glance, `restartWorkspaceHandler` is not consistent with `startWorkspace()`

Reply:
I've renamed the handler, 
thank you, Vitaliy!